#from вар2 import DataBase
#try:
 #   DataBase.callFunction('set_model', matA, 'pink')
#except Exception as error:
#    print(str(error))